### ISSUE TITLE
ReferenceError: Desmos is not defined

### ISSUE THREAD

**Issue by danielfostiak** [ysulyma/desmos-react#1]
After installing desmos-react with npm, I am presented with this error when including the Usage code in my App.js react app. 
Module../node_modules/desmos-react/index.js
node_modules/desmos-react/index.js:135
  132 |   return forwardRef(component);
  133 | }
  134 | 
> 135 | export var FourFunctionCalculator = makeBasicCalculator(Desmos.FourFunctionCalculator, "FourFunctionCalculator");
  136 | export var ScientificCalculator = makeBasicCalculator(Desmos.ScientificCalculator, "ScientificCalculator");
  137 | export function useHelperExpression(opts) {
  138 |   var calculator = useContext(DesmosContext);

**Comment by ysulyma** [comment]
You need to include Desmos in your page:
```html
<script src="https://www.desmos.com/api/v1.6/calculator.js?apiKey=dcb31709b452b1cf9dc26972add0fda6"></script>
```

**Comment by danielfostiak** [comment]
> You need to include Desmos in your page:
> 
> ```
> <script src="https://www.desmos.com/api/v1.6/calculator.js?apiKey=dcb31709b452b1cf9dc26972add0fda6"></script>
> ```

Thank you, all I had to do was add that to my index.html. However I am now at a new problem, I am trying to make the demo work but all I am getting is what seems to be a line appearing. This is what I have used as the demo function: 
<img width="592" alt="image" src="https://user-images.githubusercontent.com/37837102/160240791-5871400d-84a0-4b7c-9226-b80bc26221ca.png">
and this is all I am getting rendered:
<img width="721" alt="image" src="https://user-images.githubusercontent.com/37837102/160240816-79e7d762-bc92-468a-9158-6ea277c94a2e.png">

also, im sorry if these questions are very obvious or dumb, this is my first time using a react wrapper. any help would be appreciated.
